src/internal/rest/group.go

```diff
@@ -0,0 +1,33 @@
+package rest
+
+import (
+	domainGroup "github.com/aldinokemal/go-whatsapp-web-multidevice/domains/group"
+	"github.com/aldinokemal/go-whatsapp-web-multidevice/pkg/utils"
+	"github.com/gofiber/fiber/v2"
+)
+
+type Group struct {
+	Service domainGroup.IGroupService
+}
+
+func InitRestGroup(app *fiber.App, service domainGroup.IGroupService) Group {
+	rest := Group{Service: service}
+	app.Post("/group/join_with_link", rest.JoinGroupWithLink)
```

Comment:
I prefer `/group/join-with-link` instead of `/group/join_with_link`

Reply:
Thanks, didn't know what the convention was as there weren't any links composed of multiple words yet, I changed it.